schema.graphql

```diff
@@ -63,6 +63,9 @@ type Market @entity {
     positions: [Position!]! @derivedFrom(field: "marketRef")
     "open orders"
     openOrders: [OpenOrder!]! @derivedFrom(field: "marketRef")
+
+    "trader open interest"
+    traderOpenInterest: BigDecimal!
```

Comment:
prefer to call it `takerOpenInterest`. Like we use `takerPositionSize` in `TraderMarket` to note it is for taker only.